cluster/src/test/java/com/hartwig/pipeline/tertiary/lilac/LilacBamSlicerTest.java

```diff
@@ -31,9 +31,9 @@ protected Stage<LilacBamSliceOutput, SomaticRunMetadata> createVictim() {
     @Override
     protected List<String> expectedCommands() {
         return List.of(
-                "/opt/tools/sambamba/0.6.8/sambamba slice -L /opt/resources/lilac/37/hla.37.bed -o /data/output/reference.hla.bam /data/input/reference.bam",
+                "/opt/tools/samtools/1.14/samtools view -f bam -L /opt/resources/lilac/37/hla.37.bed -b -o /data/output/reference.hla.bam /data/input/reference.bam",
```

Comment:
Maybe our current framework would make it too much trouble but it would be really great to test the conditional logic that decides which argument to pass for the input file type.